### ISSUE TITLE
Possible bug: __repr__ method of Run class overwrites .game and .category variables

### ISSUE THREAD

**Issue by thecnoNSMB** [blha303/srcomapi#13]
.game and .category, before the object is printed, refer to ID strings. This behavior is inconsistent, especially when working between the interpreter and scripting.

**Comment by thecnoNSMB** [comment]
Just skimming over the code, it looks like the Leaderboard class has the same problem.

**Comment by blha303** [comment]
Yeah, I think I went with a method that solved a bug I had at the time, but turning data into a class instance definitely shouldn't be happening in repr.
